src/components/Create/components/RewardsList/AddEditRewardModal.tsx

```diff
@@ -44,7 +44,8 @@ interface AddEditRewardModalFormProps {
   externalUrl?: string | undefined
 }
 
-const NFT_FILE_UPLOAD_EXTRA = t`NFT will be cropped to a 1:1 square in thumbnail previews on the Juicebox app.`
+const NFT_FILE_UPLOAD_EXTRA = t`Images will be cropped to a 1:1 square in thumbnail previews on the Juicebox app.`
+const MAX_NFT_FILE_SIZE_MB = 100
```

Comment:
@johnnyd-eth shit this actually needs to be `100 * 1024 * 1024` apparently. pls confirm

Reply:
Nup, as is is good. We reduce the fileSize in later checks:

<img width="553" alt="Screen Shot 2023-02-21 at 9 01 49 pm" src="https://user-images.githubusercontent.com/96150256/220327412-946acc94-aa3f-48dc-8e6c-563c8d571e90.png">

Have changed the `sizeLimit` prop to `sizeLimitMB` as suggested in Discord.